internal/dag/dag.go

```diff
@@ -373,9 +388,10 @@ func (d *DAG) recompute() dag {
 			}
 		}
 
-		visited := make(map[meta]bool)
+		var visited []*ingressroutev1.IngressRoute
```

Comment:
this can be written as
```
ve := d.processIngressRoute(ir, prefixMatch, nil, host, service, vhost, orphaned)
```
because the value of `visited` before and after the call to processIngressRoute is unchanged.

Reply:
Ah nice - Thanks!